### DISCUSSION TITLE
Display cross sales cart

### DISCUSSION THREAD

**Discussion by bros992** [woocommerce/woocommerce#45039] [WooCommerce, Blocks, and Site Editing]
## What do you think is missing from the Cart & Checkout blocks?

Hello, thank you for your plugin and widgets. I would like to display cross sales products in cart page. That would be very helpful. Thank you

**Comment by nerrad** [comment]
HI there @bros992  👋🏻

Thanks so much for taking the time to leave some feedback - much appreciated! We make note of all feedback we receive and we also close issues with the feedback as active issues are used for bugs or feature/enhancements. It does not mean that we don't note or review the feedback. Closed issues can also still be commented on.
